src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,48 +1,64 @@
 import React from 'react';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
 
-export const TodoModal: React.FC = () => {
+type Props = {
+  user: User | null;
+  todo: Todo | null;
+  onClose: () => void;
+};
+
+export const TodoModal: React.FC<Props> = ({
+  user,
+  todo,
+  onClose,
+}) => {
   return (
     <div className="modal is-active" data-cy="modal">
       <div className="modal-background" />
 
-      {true ? (
-        <Loader />
-      ) : (
+      {user && todo ? (
         <div className="modal-card">
           <header className="modal-card-head">
             <div
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              {`Todo #${todo?.id}`}
```

Comment:
remove it in all cases
```suggestion
              {`Todo #${todo.id}`}
```